libs/libcommon/src/libcommon/constants.py

```diff
@@ -62,3 +62,21 @@
 
 CROISSANT_MAX_CONFIGS = 100
 MAX_NUM_ROWS_PER_PAGE = 100
+
+DESCRIPTIVE_STATISTICS_PROMETHEUS_HISTOGRAM_BUCKETS = (
```

Comment:
the buckets are a bit unbalanced. What if we do a generic list that could be used for any worker, with an exponential increase?

Such as:

```
0.01 / 0.02 / 0.05 / 0.1 / 0.2 / 0.5 / 1 / 2 / 5 / 10 / 30 / 60 / 60*2 / 60*5 / 60*10 / 60*20 / 60*60?
```

Reply:
and let's call it WORKER_PROMETHEUS_HISTOGRAM_BUCKETS?

Reply:
ok, too late ....

Reply:
Or, to keep closer to the [default values in prometheus](https://github.com/prometheus/client_python/blob/7a80f001237fe881d3607861947292abc85bf205/prometheus_client/metrics.py#L586):

```
WORKER_PROMETHEUS_HISTOGRAM_BUCKETS = (.005, .01, .025, .05, .075, .1, .25, .5, .75, 1.0, 2.5, 5.0, 7.5, 10.0, 30.0, 60.0, 2.5*60.0, 5*60.0, 7.5*60.0, 10*60.0, 30*60.0, 60*60.0, INF)
```

Reply:
^ no need to tweak for minutes/hours, in fact... these boundaries will be used to compute statistics. Let's use the same space between steps, until reaching the max value we want (5000 = 1h23minutes is ok, as we stop the job runner after 40 minutes)